schema.graphql

```diff
@@ -662,117 +669,19 @@ type Disbursed @entity {
     timestamp: BigInt!
 }
 
-# Trader Historical Data
-
-type TraderDayData @entity {
```

Comment:
Just checked with Dylan. The frontend is still using this schema for vePERP referral. It can not be removed.